setup.py

```diff
@@ -54,6 +54,7 @@
     ],
     extras_require={
         "pecos": ["pytket-pecos ~= 0.1.18"],
+        "calendar": ["matplotlib ~= 3.8.3", "pandas ~= 2.2.1"],
```

Comment:
Documentation about this functionality and how to enable it should be added to the front documentation page.

Reply:
Added to the front page as requested